### PR TITLE
Better, more distinguishable colors

### PR DESCRIPTION
**Pull request by matejos**
Hello! 
First of all, I love your extension, great work! But one thing has been bothering me and that is that usually the party colors are very similar to each other because of the randomized generation. So I tried to fix that, as we know that there will be max of 10 parties (if everyone is queuing solo), we can make an array of 10 well distinguishable colors and just pick from them. Here you can have a look:
[Before](https://cloud.githubusercontent.com/assets/10269067/19730683/4ce5c796-9b9b-11e6-845e-690395a19e51.png) - [After](https://cloud.githubusercontent.com/assets/10269067/19730685/4d0609b6-9b9b-11e6-93da-76b2f2c01402.png)

### CODE REVIEW

**Comment by nuller1joe** [comment]
To respond to the last commits question, i removed all colors between 270 and 360 because they are all pinkish.. We are men, no pink for us ;)

I like your idea, but the colors are awful IMO..  I think you should use other saturation and value numbers like: HSVtoRGB(inRange , 0.9, 0.6); or even HSVtoRGB(inRange , 0.4, 0.6);

I think the perfect solution would still be to randomize colors, but making sure that there are atleast 35 hue between each color.. So team memeber gets a random color, lets say 27. The next is 69 which is fine, the next is 73 which is no good. So this guys gets his color rounded up to 69+35 = 104. If then someone has the random number 43, he will get 104+35= 139 and so on and so on.. I know this a alot more work than needed, but IMO would be the programmers dream

**Comment by matejos** [comment]
Ad 1:
Okay I understand the reason now, but then if some party gets for example H30 and some other party gets H300, they both end up having the same color H30... (btw you could have just passed hash % 270)

Ad 2:
[Alright, is HSVtoRGB(inRange , 0.9, 0.6); good?](https://cloud.githubusercontent.com/assets/10269067/19773157/c6635f66-9c68-11e6-8e8a-25de722c4e95.png)

Ad 3:
I don't mean to be rude, but that would be the programmer's nightmare imo. Why do you insist on doing unnecessary ineffective randomization cycles? We know there can be max of 10 parties so we can have 10 perfectly different static colors. That's the optimal, lightweight solution.

**Comment by nuller1joe** [comment]
1: You are right, should be fixed

2: IMO much better

3: Yea, let me rephrase it to the UX guys dream then... A little boring that its always the same colors that shows.. :)

**Comment by matejos** [comment]
2: Ok so I commited those values.
3: Oh ok I understand now :D Boring maybe... maybe... but fast! :D

**Comment by Poheart** [comment]
I don't have much opinions on this topics, I think both solution are okay, as long as it with least  performance impact  ;)

**Comment by matejos** [comment]
@Poheart Could you push new version? :)

**Comment by Poheart** [comment]
Will push when i have worked on the fix request from Opera/Firefox
